Windows x86 failed with:

```
FAIL: test_terminate (test.test_multiprocessing_spawn.test_threads.WithThreadsTestPool.test_terminate)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\cpython\cpython\Lib\test\_test_multiprocessing.py", line 2688, in test_terminate
    self.assertLess(join.elapsed, 2.0)
AssertionError: 27.2650000000001 not less than 2.0
```

Ubuntu failed with:

```
FAIL: test_terminate (test.test_multiprocessing_spawn.test_threads.WithThreadsTestPool.test_terminate)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/runner/work/cpython/cpython-ro-srcdir/Lib/test/_test_multiprocessing.py", line 2688, in test_terminate
    self.assertLess(join.elapsed, 2.0)
AssertionError: 27.996608135999963 not less than 2.0
```

Surprise surprise, macOS failed with:

```
FAIL: test_terminate (test.test_multiprocessing_spawn.test_threads.WithThreadsTestPool.test_terminate)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/runner/work/cpython/cpython/Lib/test/_test_multiprocessing.py", line 2688, in test_terminate
    self.assertLess(join.elapsed, 2.0)
AssertionError: 27.589956145999963 not less than 2.0
```

--- a/Lib/test/_test_multiprocessing.py
+++ b/Lib/test/_test_multiprocessing.py
@@ -2578,7 +2578,7 @@ class _TestPool(BaseTestCase):
         self.assertTimingAlmostEqual(get.elapsed, TIMEOUT1)
 
     def test_async_timeout(self):
-        res = self.pool.apply_async(sqr, (6, TIMEOUT2 + 1.0))
+        res = self.pool.apply_async(sqr, (6, TIMEOUT2 + support.SHORT_TIMEOUT))
         get = TimingWrapper(res.get)
         self.assertRaises(multiprocessing.TimeoutError, get, timeout=TIMEOUT2)
         self.assertTimingAlmostEqual(get.elapsed, TIMEOUT2)

